target_chains/solana/programs/pyth-solana-receiver/src/lib.rs

```diff
@@ -254,6 +254,49 @@ pub mod pyth_solana_receiver {
     pub fn reclaim_rent(_ctx: Context<ReclaimRent>) -> Result<()> {
         Ok(())
     }
+    
+    // New function to initialize a PriceUpdateV2 account without verifications (for testing)
+    pub fn initialize_price_update_v2(
```

Comment:
The stub needs to be a different program in `programs`, otherwise there's a risk we deploy this

Reply:
ex: 
`programs/pyth-solana-receiver-stub`

Reply:
I see. Will take care of this. Thanks for informing me.